photon-ml/src/integTest/scala/com/linkedin/photon/ml/algorithm/RandomEffectCoordinateTest.scala

```diff
@@ -55,17 +58,17 @@ class RandomEffectCoordinateTest extends SparkTestUtils with GameTestUtils {
   @Test(dataProvider = "numEntitiesDataProvider")
   def testScore(numEntities: Int): Unit = sparkTest("testScore") {
     val (coordinate, model) = generateRandomEffectCoordinateAndModel(
-      randomEffectId, featureShardId, numEntities, NumTrainingSamples, Dimensionality)
+      randomEffectId, featureShardId, numEntities, NUM_TRAINING_SAMPLES, DIMENSIONALITY)
 
     val score = coordinate.score(model)
 
-    assertEquals(score.scores.count, numEntities * NumTrainingSamples)
+    assertEquals(score.scores.count, numEntities * NUM_TRAINING_SAMPLES)
     assertTrue(score.scores.map(_._2).collect.forall(_ == 0.0))
   }
 }
 
 object RandomEffectCoordinateTest {
-  val NumTrainingSamples = 1000
-  val Dimensionality = 10
-  val NumPartitions = 4
+  val NUM_TRAINING_SAMPLES = 1000
+  val DIMENSIONALITY = 10
+  val NUM_PARTITIONS = 4
```

Comment:
I wasn't aware. We should stick to the Scala style guide. I'll definitely change these back (as well as any other such changes in the code) just so we don't forget, in case we end up doing it piecemeal (as with the author tags).